oracle/provider/binance.go

```diff
@@ -199,31 +199,51 @@ func (p *BinanceProvider) SubscribeCurrencyPairs(cps ...types.CurrencyPair) erro
 func (p *BinanceProvider) GetTickerPrices(pairs ...types.CurrencyPair) (map[string]types.TickerPrice, error) {
 	tickerPrices := make(map[string]types.TickerPrice, len(pairs))
 
+	tickerErrs := 0
 	for _, cp := range pairs {
 		key := cp.String()
 		price, err := p.getTickerPrice(key)
 		if err != nil {
-			return nil, err
+			p.logger.Warn().Err(err)
```

Comment:
Let's add a specific error type for the tickers & candles here, so when we see the log we know where it is

Reply:
The `ErrTickerNotFound` with the endpoint and pair is what gets logged here already (thats the error that gets returned from `p.getTickerPrice`)